chimney/src/main/scala-2/io/scalaland/chimney/internal/compiletime/ConfigurationsPlatform.scala

```diff
@@ -104,65 +86,81 @@ private[compiletime] trait ConfigurationsPlatform extends Configurations { this:
 
     // TODO: adjust for new config type
```

Comment:
Are these TODO comments still relevant?

For tailrec I believe we agreed to leave it for now, but how about the other one?

Reply:
I think this TODO was overlooked during cleanup, it can be removed

Reply:
removed ✅